integration/ecr_auth.go

```diff
@@ -0,0 +1,21 @@
+package integration
+
+import (
+	"encoding/json"
+	"os"
+)
+
+type ecrCredentialsFile struct {
```

Comment:
nit: camelCase for JSON. Also why are the credential fields not private? Do they need to be accessible somewhere else?

Reply:
They have to be public to be able to unmarshal them.